### PR TITLE
Fix create+prepare rt kernel scripts

### PR DESCRIPTION
**Pull request by Athen-Player1**
## Proposed changes

Please explain the changes you made. Add pictures or videos to explain them better if appropriate.

_added libelf-dev library to the install/ added --no-cert check to the wget commands / added fix for no cert build error_

## Checklist

Before posting a PR ensure that from each of the below categories **AT LEAST ONE BOX HAS BEEN CHECKED**. If more than one category is applicable then more can be checked. Also ensure that the proposed changes have been filled out with relevant information for reviewers.

## Tests

- [ ] No tests required to be added. (For small changes that will be tested by CI/CD infrastructure).
- [ ] Added/Modified automated and PhantomHand CI tests (if a new class is added (Python or C++), the interface of that class must be unit tested).
- [ ] Manually tested in simulation (if simulation specific or no hardware required to test the functionality). 
- [x] Manually tested on hardware (if hardware specific or related).

## Documentation

- [x] No documentation required to be added.
- [ ] Added documentation (For any new feature, explain what it does and how to use it. Write the documentation in a relevant space, e.g. Github, Confluence, etc).
- [ ] Updated documentation (For changes to pre-existing features mentioned in the documentation).

### CODE REVIEW

File: bin/prepare_rt_kernel.sh [hunk 1]
```diff
@@ -18,6 +19,9 @@ make mrproper
 cp /boot/config-`uname -r` .config
 
 make olddefconfig
+# Setting systemkeys to avoid; No rule to make target 'debian/certs/test-signing-certs.pem' build error
+scripts/config --set-str SYSTEM_TRUSTED_KEYS ""
```
Comment (toliver) [inline bin/prepare_rt_kernel.sh:23]: Where does this `scripts/config` come from? (I don't have it in my machine)

**Review by toliver — CHANGES_REQUESTED** [review]

File: bin/prepare_rt_kernel.sh (hunk 1 above)
Comment (Athen-Player1) [reply bin/prepare_rt_kernel.sh:23]: its in the downloaded kernel folder ~/linux-*.*.*/scipts

File: bin/prepare_rt_kernel.sh [hunk 2]
```diff
@@ -5,8 +5,9 @@ RT_PATCH_VERSION="4.13.13-rt5"
 KERNEL_DIR="v4.x"
 KERNEL_VERSION="4.13.13"
```
Comment (dg-shadow) [inline bin/prepare_rt_kernel.sh:6]: Should these be "X.X.X" as for other script? Or make a seperate file that is sourced into both?

**Review by dg-shadow — CHANGES_REQUESTED** [review]

File: bin/prepare_rt_kernel.sh (hunk 1 above)
Comment (toliver) [reply bin/prepare_rt_kernel.sh:23]: Oh, of course :sweat_smile:

**Review by toliver — APPROVED** [review]

**Review by dg-shadow — APPROVED** [review]

**Pull request merged**